src/arc.ts

```diff
@@ -127,7 +127,7 @@ export function checkNetwork() {
     }
 
     // check if Metamask account access is enabled, and if not, ask to enable it
-    enableMetamask();
+    // enableMetamask();
```

Comment:
Why is this commented out now?

Reply:
it is commented out because like this it would pop up a metamask dialog on loading the page, and I was in doubt on wether that is desired behavior yes or no. 

So I made a new issue with specs: https://github.com/daostack/alchemy/issues/493  According to these specs, we should _not_ call "enableMetamask" here. But before makign the definitive decision, I'd like some feedback on that issue, both from you @tibetsprague as wella s from @A-Zak

Reply:
ímplemented the specs